build.gradle.kts

```diff
@@ -31,14 +31,4 @@ val buildVersion = "2.6.0-rc2"
 val snapshot = System.getenv("SNAPSHOT_BUILD") ?: ""
 version = "${buildVersion}${snapshot}"
 group = "cloud.genesys"
-
```

Comment:
Is commenting out nexusPublishing intentional?

Reply:
I had to comment that out to get my version to run. I've fixed it now.